.prow/scripts/test-end-to-end.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+if ! cat /etc/*release | grep -q stretch; then
+    echo ${BASH_SOURCE} only supports Debian stretch. 
+    echo Please change your operating system to use this script.
+    exit 1
+fi
+
+echo "
+This script will run end-to-end tests for Feast Core and Online Serving.
+
+1. Install Redis as the store for Feast Online Serving.
+2. Install Postgres for persisting Feast metadata.
+3. Install Kafka and Zookeeper as the Source in Feast.
+4. Install Python 3.7.4, Feast Python SDK and run end-to-end tests from 
+   tests/e2e via pytest.
+"
+
+echo "
+============================================================
+Installing Redis at localhost:6379
+============================================================
+"
+apt-get -qq update
+# Allow starting serving in this Maven Docker image. Default set to not allowed.
+echo "exit 0" > /usr/sbin/policy-rc.d
+apt-get -y install redis-server wget > /var/log/redis.install.log
+redis-server --daemonize yes
+redis-cli ping
+
+echo "
+============================================================
+Installing Postgres at localhost:5432
+============================================================
+"
+apt-get -y install postgresql > /var/log/postgresql.install.log
+service postgresql start
+# Initialize with database: 'postgres', user: 'postgres', password: 'password'
+cat <<EOF > /tmp/update-postgres-role.sh
+psql -c "ALTER USER postgres PASSWORD 'password';"
+EOF
+chmod +x /tmp/update-postgres-role.sh
+su -s /bin/bash -c /tmp/update-postgres-role.sh postgres
+export PGPASSWORD=password
+pg_isready
+
+echo "
+============================================================
+Installing Zookeeper at localhost:2181
+Installing Kafka at localhost:9092
+============================================================
+"
+wget -qO- https://www-eu.apache.org/dist/kafka/2.3.0/kafka_2.12-2.3.0.tgz | tar xz
+mv kafka_2.12-2.3.0/ /tmp/kafka
+nohup /tmp/kafka/bin/zookeeper-server-start.sh /tmp/kafka/config/zookeeper.properties &> /var/log/zookeeper.log 2>&1 &
+sleep 5
+tail -n10 /var/log/zookeeper.log
+nohup /tmp/kafka/bin/kafka-server-start.sh /tmp/kafka/config/server.properties &> /var/log/kafka.log 2>&1 &
+sleep 5
+tail -n10 /var/log/kafka.log
+
+echo "
+============================================================
+Building jars for Feast
+============================================================
+"
+
+.prow/scripts/download-maven-cache.sh \
+    --archive-uri gs://feast-templocation-kf-feast/.m2.2019-10-24.tar \
+    --output-dir /root/
+
+# Build jars for Feast
+mvn --quiet --batch-mode --define skipTests=true clean package
+
+echo "
+============================================================
+Starting Feast Core
+============================================================
+"
+# Start Feast Core in background
+cat <<EOF > /tmp/core.application.yml
+grpc:
```

Comment:
Do we want to leave this yml file here? It seems like it will go stale eventually, but I guess it is ok as a start.

Reply:
Yes we can move the test scripts and the resources (like yaml configuration) into tests folder or infra folder as an improvement